apps/cloud/src/api/user.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Created by acunningham on 22/05/17.
+ */
+import Store from '@raincatcher/store';
+import webapi, {ApiService, StoreApiService, WebApiConfig}  from '@raincatcher/webapi';
+import * as express from 'express';
+
+
+// Define User datatype
+interface User {
+  id: string,
+  username: string,
+  name: string,
+  position: string,
+  phone: string,
+  email: string,
+  notes: string,
+  avatar: string,
+  banner: string,
```

Comment:
Maybe it would also make sense to have a separate `Profile` entity to store data that's not strictly required from an auth point of view.

That way users could:

- Replace it completely with their own intended set of fields
- Use a separate storage engine just for that data